### PR TITLE
Feature/product page

### PR DESCRIPTION
**Pull request by OlecsandrStelmakh**
https://b24-lmffg5.bitrix24.ua/workgroups/group/8/tasks/task/view/58/

https://2222qqqq.000webhostapp.com/
https://2222qqqq.000webhostapp.com/?drinks_125

### CODE REVIEW

File: css/product-page.css [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  height: 560px;
+}
+
+.product-page-description {
+  width: inherit;
+  padding: 0 20px;
+}
+
+.product-page-name {
+  font-size: 2em;
```
Comment (maradroman) [inline css/product-page.css]: must be rem not em

File: css/product-page.css [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+.product-page-order-inform {
+  border-color: #d4d4d4;
+  border-bottom-style: solid;
+  border-top-style: solid;
+  border-width: 1px;
+  margin: 20px 0;
+}
+.product-page-price {
+  font-size: 1.5em;
```
Comment (maradroman) [inline css/product-page.css]: must be rem not em

File: css/product-page.css [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+  text-transform: uppercase;
+}
+
+.product-page-order-conteiner {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+}
+
+.product-page-button-conteiner {
```
Comment (maradroman) [inline css/product-page.css]: container*

File: css/product-page.css [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+  margin: 0;
+}
+
+.product-properties--color,
+.product-properties--style,
+.product-properties--region,
+.product-properties--brand,
+.product-properties--manufacturer,
+.product-properties--sort {
+  color: goldenrod;
```
Comment (maradroman) [inline css/product-page.css]: color must be in HEX, #daa520 expected

File: css/product-page.css [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+  margin: 0;
+}
+
+.product-page-order-inform {
+  border-color: #d4d4d4;
+  border-bottom-style: solid;
+  border-top-style: solid;
+  border-width: 1px;
+  margin: 20px 0;
+}
```
Comment (maradroman) [inline css/product-page.css:44]: space

**Comment by OlecsandrStelmakh** [comment]
bug fixes + now everything works in IE

File: .vscode/settings.json [hunk 6]
```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": true
```
Comment (maradroman) [inline .vscode/settings.json]: add vscode to gitignore

File: css/header.css [hunk 7]
```diff
@@ -147,7 +149,7 @@
 
 .header-mid-basket-quantity {
   position: absolute;
-  font-size: 0.8em;
+  font-size: 0.8rem;
```
Comment (maradroman) [inline css/header.css]: .8rem

**Review by maradroman — DISMISSED** [review]
remove mentioned (vscode and leading zero) and merge into dev

**Review by maradroman — APPROVED** [review]

**Pull request merged**